spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -2757,6 +2757,20 @@
       "name": "spring.session.redis.flush-mode",
       "defaultValue": "on-save"
     },
+    {
+      "name": "spring.session.redis.repository",
+      "description": "Redis session repository implementation to use.",
+      "values": [
+        {
+          "value": "default",
+          "description": "Use default Redis session repository."
+        },
+        {
+          "value": "indexed",
```

Comment:
OTOH, `RedisIndexedSessionRepository` makes this quite straightforward. One might get tempted to throw mention of _events_ here as well (based on soon to be removed `EnableRedisHttpSession#enableIndexingAndEvents`) however I'd avoid that as support for session events is an implementation detail of a session repository (and not something expressed through a contract, like indexing capabilities).